### ISSUE TITLE
Specifying a tool maintainer in `.shed.yml` files

### ISSUE THREAD

**Issue by simonbray** [galaxy-iuc/standards#65]
For the autoupdate PRs (see https://github.com/galaxyproject/tools-iuc/issues/3533) it would be nice to be able to ping one or more tool maintainers to review the changes. This would require a way to associate GitHub users as maintainers with a tool, similar to how they are listed in bioconda recipes (e.g. https://github.com/bioconda/bioconda-recipes/blob/master/recipes/spades/meta.yaml#L73). I assume the most sensible way is to add a field to the `.shed.yml` files.

I am wondering, firstly if there would be any objections, and secondly, if anyone has any tips on implementing it? From the point of view of the CI it would be adequate to just add a `maintainers` field to the YAML files, which can be read whenever a new PR is opened. On the other hand it might be preferable to integrate this field directly into the Galaxy code. In the latter case, I guess the correct place to start would be https://github.com/galaxyproject/galaxy/blob/dev/lib/tool_shed/webapp/model/__init__.py#L182?

(hit submit too soon)

**Comment by nsoranzo** [comment]
Maybe it's easier to use a [CODEOWNERS](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners) file? The [Training Material](https://github.com/galaxyproject/training-material/) repo uses it.

**Comment by simonbray** [comment]
That's an very interesting idea and probably would indeed be a lot easier, thanks @nsoranzo!

**Comment by hexylena** [comment]
yeah, I already use it for jbrowse+circos, the rest of IUC is muted for me to make the signal:noise ratio manageable. https://github.com/galaxyproject/tools-iuc/blob/master/.github/CODEOWNERS

Fully support this idea @simonbray :)
